_posts/2021-02-18-survey/survey.Rmd

```diff
@@ -0,0 +1,314 @@
+
+Switching to the topic of `tensorflow`'s Python dependencies causing problems with installation, my experience (from GitHub issues, plus adding in my own) has been that difficulties are quite system-dependent. On some OSes, complications seem to appear more often than on others; and low-control (to the individual user) environments like HPC clusters can make things especially difficult. In any case though, I have to (unfortunately) admit that when installation problems appear, they can be very tricky to solve.
+
+On to the next point (and I promise, this was by far the longest subsection).
+
+##### `tidymodels` integration
+
+The second most-frequent mention clearly was the wish for tighter `tidymodels` integration. Here, we wholeheartedly agree. As of today, there is no automated way to accomplish this for `torch` models generically, but it can be done for specific model implementations.
+
+Last week, [torch, tidymodels, and high-energy physics](https://blogs.rstudio.com/ai/posts/2021-02-11-tabnet/) featured the first `tidymodels`-integrated `torch` package. And there's more to come. In fact, if you are developing a package in the `torch` ecosystem, why not consider doing the same? Should you run into problems, the growing `torch` community will be happy to help.
+
+##### Documentation, examples, teaching materials
+
+Thirdly, several respondents expressed the wish for more documentation, examples, and teaching materials. Let me say that here, the situation is different for TensorFlow than for `torch`.
+
+For [`tensorflow`](https://tensorflow.rstudio.com/), the website has a multitude of guides, tutorials, and examples. For [`torch`](torch.mlverse.org), reflecting the discrepancy in respective lifecycles, materials are not that abundant (yet). However, after a recent refactoring, the website has a new, four-part [Get started](https://torch.mlverse.org/start/) section addressed to both beginners in DL and experienced TensorFlow users curious to learn about `torch`. After this hands-on introduction, a good place to get more technical background would be the section on [tensors, autograd, and neural network modules](https://torch.mlverse.org/technical/).
+
+Truth be told, though, nothing would be more helpful here than contributions from the community. Whenever you solve even the tiniest (which is often how things appear to oneself) problem, consider creating a vignette explaining what you did. Future users will be thankful, and a growing user base means that over time, it'll be your turn to find that some things have already been solved for you!
+
+Which leads me to the last -- aggregate -- point.
```

Comment:
Again, I think you can get rid of this sentence; that one section leads to another is sufficiently self-evident from the way it's written.

Reply:
ok